vscode/src/edit/provider.ts

```diff
@@ -201,7 +205,7 @@ export class EditProvider {
             const editTimeToFirstTokenSpan = tracer.startSpan('cody.edit.provider.timeToFirstToken')
             const model = this.config.task.model
             const contextWindow = modelsService.getContextWindowByID(model)
-            const versions = await currentSiteVersion()
+            const versions = this.siteVersion ?? (await currentSiteVersion())
```

Comment:
```suggestion
            const versions = this.siteVersion ?? (await currentSiteVersion())
```

do we need to store the site version if we only need the codyAPIVerion?

i was also looking at the site version file, maybe you could use [this function ](https://sourcegraph.sourcegraph.com/github.com/sourcegraph/cody/-/blob/lib/shared/src/sourcegraph-api/siteVersion.ts?L34-36)to get the codyAPIVersion instead?

Reply:
I think we need both the siteVersion and codyApiVersion. The earlier implementation fetched both `await currentSiteVersion()`.